ADAL/tests/unit/ADUserInformationTests.m

```diff
@@ -41,14 +41,141 @@ - (void)tearDown
     [super tearDown];
 }
 
+#pragma mark - isEqual
+
+- (void)testIsEqual_whenAllPropertiesAreEqual_shouldReturnTrue
+{
+    ADUserInformation *lhs = [self adCreateUserInformation:@"eric_cartman@contoso.com"];
+    ADUserInformation *rhs = [self adCreateUserInformation:@"eric_cartman@contoso.com"];
+    
+    XCTAssertEqualObjects(lhs, rhs);
+}
+
+- (void)testIsEqual_whenAllClaimsIsNotEqual_shouldReturnFalse
+{
+    ADUserInformation *lhs = [self createEmptyUserInformation];
```

Comment:
Same thing here, some of the ADUserInformation properties might be nil and I would like that to be tested to at least some extent.